Header header
geometry_msgs/Vector3 tc_o
geometry_msgs/Vector3 tcstar_o
geometry_msgs/Vector3 theta_mu
